Description: cutils/atomic.h uses stdatomic.h which is not supported by C++11
Author: Kai-Chung Yan <seamlikok@gmail.com>
Last-Update: 2016-09-18
--- a/include/utils/StrongPointer.h
+++ b/include/utils/StrongPointer.h
@@ -17,7 +17,9 @@
 #ifndef ANDROID_STRONG_POINTER_H
 #define ANDROID_STRONG_POINTER_H
 
-#include <cutils/atomic.h>
+extern "C" {
+#include "cutils_atomic.h"
+}
 
 #include <stdint.h>
 #include <sys/types.h>
--- a/include/utils/Atomic.h
+++ b/include/utils/Atomic.h
@@ -17,6 +17,8 @@
 #ifndef ANDROID_UTILS_ATOMIC_H
 #define ANDROID_UTILS_ATOMIC_H
 
-#include <cutils/atomic.h>
+extern "C" {
+#include "cutils_atomic.h"
+}
 
 #endif // ANDROID_UTILS_ATOMIC_H
